keras_rcnn/models.py

```diff
@@ -1,33 +1,40 @@
 # -*- coding: utf-8 -*-
 
 import keras
+import keras_resnet
 import keras_rcnn.layers
+import keras_rcnn.heads
 
 
 class RCNN(keras.models.Model):
```

Comment:
This looks great!

We should also consider adding a `features` argument that users would use to pass their feature extractor (e.g. ResNet50). What do you think?